hivemind/moe/server/server.py

```diff
@@ -332,7 +319,7 @@ def background_server(*args, shutdown_timeout=5, **kwargs) -> Tuple[Endpoint, Li
         if runner.is_alive():
             logger.info("Server failed to shutdown gracefully, terminating it the hard way...")
             runner.kill()
-            logger.info("Server terminated")
+            logger.info("Server terminated.")
```

Comment:
nit: We usually don't use trailing dots in the log messages (e.g., see #419). Please remove them.